public/api/addContact.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+$inData = getRequestInfo();
+
+$first_name = $inData["first name"];
+$last_name = $inData["last_name"];
+$userId = $inData["userId"];
+
+$database_url = parse_url(getenv('DATABASE_URL'));
+
+$server = $database_url['host'];
+$user = $database_url['user'];
+$pass = $database_url['pass'];
+$db = substr($database_url['path'], 1);
+
+$conn = new mysqli($server, $user, $pass, $db);
+
+if ($conn->connect_error) 
+{
+	returnWithError($conn->connect_error);
+} else 
+{
+	$stmt = conn->prepare("INSERT into contacts (UserID, first, last) Values(?, ?, ?)");
```

Comment:
we need to use the ERD to specify the table column names

Reply:
contacts also have a lot more properties than juse user_id, first_name, and last_name

Reply:
I'm a little confused as to how to go about this, not sure where I can see the ERD. Also I'm unsure about the properties to be included.

Reply:
![image](https://github.com/user-attachments/assets/8aba5d99-3962-4efe-b9ea-c27db3459672)